app/helpers/application_helper/button/dialog.rb

```diff
@@ -1,4 +1,4 @@
-class ApplicationHelper::Button::Dialog < ApplicationHelper::Button::Basic
+class ApplicationHelper::Button::Dialog < ApplicationHelper::Button::ButtonWithoutRbacCheck
```

Comment:
This class should probably remain same and inherit from basic. Because you have action like `dialog_edit` or `dialog_new` in `miq_product_features_yml`  so they should check RBAC.